rimage/file_simple.c

```diff
@@ -356,6 +356,33 @@ int write_logs_dictionary(struct image *image)
 
 	for (i = 0; i < image->num_modules; i++) {
 		struct module *module = &image->module[i];
+		if (module->fw_ready_index > 0) {
+			Elf32_Shdr *section =
+				&module->section[module->fw_ready_index];
```

Comment:
please comment what this block is doing.

Reply:
done